_data/sites.json

```
@@ -17992,5 +17992,16 @@
         "domains": [
             "ciceksepeti.com"
         ]
+    },
+
+    {
+        "name": "OpenFoodFacts",
+        "url": "https://openfoodfacts.org",
+        "difficulty": "hard",
+        "notes": "In order to delete your OpenFoodFacts account, you need to fill out a <a href=\"https://blog.openfoodfacts.org/en/account-deletion\">form</a> and send an email.",
```

```suggestion
        "url": "https://blog.openfoodfacts.org/en/account-deletion",
        "difficulty": "hard",
        "notes": "In order to delete your OpenFoodFacts account, you need to fill out the form and send an email.",
```

We use the URL to point to the most direct deletion link possible.
Question: do you really have to fill the form AND send an e-mail? Feels so bureaucratic

---

Hi, thanks for the feedback! I tried to delete my account via the form and never got a response. I contacted the email on their page (https://world.openfoodfacts.org/terms-of-use) claiming my right to erasure (GDPR), but I haven't heard back yet.

---

How long ago did you fill the form? And when did you send the email? Just so we can establish a maximum wait time to actually describe it as impossible

---

I filled out the form a month ago, and sent 2 emails : one 2 weeks ago, and one this Monday

---

I suggest we change the difficulty to impossible then, and if you get a reply, we come back and change it back to hard.

---

Alright, if everything is good on your end, you can merge. I'll let you know if I get an answer from them! Thanks for the collaboration.